Code/Framework/AzToolsFramework/CMakeLists.txt

```diff
@@ -32,6 +32,7 @@ ly_add_target(
     COMPILE_DEFINITIONS
         PRIVATE
             $<$<CONFIG:debug>:ENABLE_UNDOCACHE_CONSISTENCY_CHECKS>
+            LY_PYTHON_SITE_PACKAGE_SUBPATH="${LY_PYTHON_VENV_SITE_PACKAGES}"
```

Comment:
I recommend using the `O3DE_` prefix instead of `LY_` going forward since the code is not Lumberyard.